other/materials_designer/create_island_defect.ipynb

```diff
@@ -0,0 +1,1079 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    materials = list(map(Material, globals()["materials_in"]))
</pre><p>Let's clean up the outputs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>